docs/theory/Validation_pipe_diameter_sizing.rst

```diff
@@ -0,0 +1,1148 @@
+
+The total investment costs, :math:`C_{install,tot}`, of the whole system are computed by:
+
+.. _eq_invest_cost:
+
+.. math::
+    :label: investment_cost
+
+    C_{invest,tot} = \\
+    C_{invest,pipe_{sup}}L + C_{invest,pipe_{ret}}L  
+    + C_{invest,demand}P_{demand,max} + C_{invest,source}P_{source,max}
+
+In which:
+
+    :math:`C_{invest,pipe_{sup}}`: the investment cost of the supply pipe(s) [€/m].
+
+    :math:`C_{invest,pipe_{ret}}`: the investment cost of the return pipe(s) [€/m].
+
+    :math:`C_{invest,demand}`: the investment cost of the heating demand(s) [€/W].
+
+    :math:`C_{invest,source}`: the investment cost of the source(s) [€/W].
+
+    :math:`L`: the length of the supply or return pipe(s) [m].
+
+    :math:`P_{demand,max}`: the maximum power specified by the demand profile(s) [W].
+
+    :math:`P_{source,max}`: the maximum power specified by the source profile(s) [W].
+
+The optimizer assumes thus that the investment costs scale with the maximum size of the assets, i.e. the profiles of the assets provided by the user.  
+This is also the case when the total heat demand of the system is lower than what the source(s) can deliver per time step.
+
+.. Specifically for demands, the optimizer routine sets the investment cost based on the maximum demand as often the size of the demand is not separately specified.
+
+The total variable operational expenditure (opex), :math:`C_{var,opex,tot}`, are the cost that depend on the operation of the asset. These cost are computed as:
+
+.. math::
+    :label: variable_opex_cost
+
+    C_{var,opex,tot} = \sum_i C_{var,opex,source} (\bar{Q}_{demand,i}+Q_{loss,i}) dt_i
+
+In which:
+
+    :math:`C_{var,opex,source}`: the variable opex cost of the source [€/MWh].
+
+    :math:`Q_{loss,i}`:  the heat loss in a pipe pair (supply plus return) during time step *i* [W].
+
+    :math:`Q_{demand,i}`: the heat delivered to the demand asset(s) at time step *i* [W].
+    
+    :math:`dt_i`:  the duration of the time step [h].
+
+Note that the variable opex is only computed based on the heat losses in the supply pipe in this version, while it should also account for the costs for the heat losses in the return lines.
+
+In this version of the optimizer routine, only the variable operational cost for sources where they scale with the thermal energy production are supported.
+For more information on the time step, see Section :ref:`subsec_V_Profiles_and_time_steps`. 
+
+.. Total heat loss, computed with final diameter?
+.. (109172.85920502 +10000000)*0.02*8760
+.. However, time steps are five days, which the user does not know. The user specifies the profile of the heat source per time step. W per hour.
+
+
+The total fixed operational expenditure (opex), :math:`C_{fixed,opex,tot}`, is computed by:
+
+.. _eq_fixed_opex_cost:
+
+.. math::
+    :label: fixed_opex_cost
+
+    C_{fixed,opex,tot} = C_{fixed,opex,source}  P_{max,source}
```

Comment:
`P_{max,source}` should be `P_{source, max}`, since this variable was already defined